README.md

```diff
@@ -64,7 +64,8 @@ environment variable
 
 ##### Steps for Windows platform
 * Download [sqlite3 sources](https://www.sqlite.org/2015/sqlite-amalgamation-3090200.zip) and extract them. Create new environment variable `SDL_SQLITE_DIR` and assign path where sqlite sources were extracted to this variable.
-* Download [sqlite3 binaries](https://www.sqlite.org/2015/sqlite-dll-win64-x64-3090200.zip). Extract them and add this folder to `PATH`.
+* Download [sqlite3 binaries](https://www.sqlite.org/2015/sqlite-dll-win64-x64-3090200.zip). Extract them to the same folder as sources and add this folder to the `PATH` environment variable.
+**Note:** It is important both sources and binaries to be in one and same directory. Name of directory is not important. It is important to be associated with `SDL_SQLITE_DIR`.
```

Comment:
@orhan-mehmedov As a suggestion:
"It is important that both sources and binaries are in one and the same directory. The name of directory is not important. It is important it is associated with `SDL_SQLITE_DIR`."